custom_components/sutro/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Sutro",
   "documentation": "https://github.com/ydogandjiev/hass-sutro",
   "issue_tracker": "https://github.com/ydogandjiev/hass-sutro/issues",
-  "dependencies": [],
+  "dependencies": ["logger"],
```

Comment:
Hey @Dexhub, can you please move this up to line 4 right before "documentation". It's making the linter unhappy in its current spot:
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/3357245/222922927-3a9ed987-4f3b-4258-9cd2-f965a4527d60.png">

Reply:
Done